ruby-challenges.rb

```diff
@@ -3,32 +3,66 @@
 
 # --------------------1) Create a method that takes in a number and determines if the number is even or odd. Use the test variables provided.
 
-reposts1 = 7
+def even_odd(number)
+    if number % 2 === 0
+        return "#{number} is even"
+    else
+        return "#{number} is odd"
+    end
+end
+
+repostsOne = 7
 # Expected output: '7 is odd'
-reposts2 = 42
+repostsTwo = 42
 # Expected output: '42 is even'
-reposts3 = 221
+repostsThree = 221
 # Expected output: '221 is odd'
 
 
-
 # -------------------2) Create a method that takes in a string and removes all the vowels from the string. Use the test variables provided. 
 # HINT: Check out this resource: https://ruby-doc.org/core-2.6/String.html#method-i-delete
 
-beatles_album1 = 'Rubber Soul'
+def remove_vowel(string)
+    if string.is_a? String
+        string.delete "aeiou"
+        else
+            "'#{string}' is not a string value."
+        end
+    end
+    
+    p remove_vowel(beatles_albumOne) # output: "Rbbr Sl"
+    p remove_vowel(beatles_albumTwo) # output: "Sgt Pppr"
+    p remove_vowel(beatles_albumThree) # output: "bby Rd"
+    p remove_vowel(5) # output: "'5' is not a string value."
+    
+    
+beatles_albumOne = 'Rubber Soul'
 # Expected output: 'Rbbr Sl'
-beatles_album2 = 'Sgt Pepper'
+beatles_albumTwo = 'Sgt Pepper'
 # Expected output: 'Sgt Pppr'
-beatles_album3 = 'Abbey Road'
+beatles_albumThree = 'Abbey Road'
 # Expected output: 'bby Rd'
 
 
 
 # -------------------3) Create a method that takes in a string and checks if the string is a palindrome. A palindrome is the same word spelled forward or backward. Use the test variables provided.
 
-palindrome_tester1 = 'Racecar'
+def palindrome(string)
+    if string.downcase == string.downcase.reverse
+      p "#{string} is a palindrome"
+    else
+      p "#{string} is not a palindrome"
+    end
+  end
+  
+palindrome(palindrome_testerOne)
+palindrome(palindrome_testerTwo)
+palindrome(palindrome_testerThree)
+
+
```

Comment:
Great job chaining methods and handling edge cases! Just make sure your indentation is uniform!